src/app/shared/player/player.component.html

```diff
@@ -12,8 +12,9 @@
     <section>
       <header>
         <div class="titles">
-          <h1 [title]="title">{{title}}</h1>
-          <h2 [title]="subtitle">{{subtitle}}</h2>
+          <h1 [title]="title"><a [href]="episodeLink" target="_top">{{title}}</a></h1>
```

Comment:
as this works now, you will get an a with no href which I think is what you're looking for, is it not?